### PR TITLE
Controller to render JSON results near a lat/lng

### PR DESCRIPTION
**Pull request by terrellt**
This is a step in the direction towards displaying all things near a user.

### CODE REVIEW

File: app/models/search_builder.rb [hunk 1]
```diff
@@ -15,4 +15,30 @@ def only_lat_longs(solr_parameters)
     solr_parameters[:fq] ||= []
     solr_parameters[:fq] << "desc_metadata__latitude_teim:[* TO *]"
   end
+
+  def distance_query(solr_parameters)
+    solr_parameters[:fq] ||= []
+    if blacklight_params[:latitude] && blacklight_params[:longitude]
+      solr_parameters[:fq] << DistanceQueryGenerator.new(*blacklight_params.slice(:latitude, :longitude, :distance).values).to_s
```
Comment (straleyb) [inline app/models/search_builder.rb]: What is the purpose of the * in front of the blacklight_params?

Comment (terrellt) [inline app/models/search_builder.rb]: Splat operator. https://endofline.wordpress.com/2011/01/21/the-strange-ruby-splat/ Turns an array into a series of arguments.

File: app/controllers/geo_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class GeoController < ApplicationController
+  include Blacklight::Base
+  copy_blacklight_config_from(CatalogController)
```
Comment (straleyb) [inline app/controllers/geo_controller.rb]: If you were to do 
```
GeoController < CatalogController
```
would that copy the config over as well?

Comment (terrellt) [inline app/controllers/geo_controller.rb]: Yes, but it also carries over all the catalog controller actions - index/show/etc.

Comment (straleyb) [inline app/controllers/geo_controller.rb]: Cool. :+1:

**Pull request merged**